Software/Development/SemesterProject/App/temperatureManager.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * temperatureManager.c
+ *
+ *  Created on: Apr 8, 2019
+ *      Author: Nicolas
+ *
+ *      The temperature manager is a module that obtains the voltage measurement
+ *      from the analogAcquisitionManager  and calculates the temperature of the sensor.
+ *      It also provides an interface for external access to the temperature.
+ *
+ *      calculateTemperature is called every second from the scheduler.
+ *
+ *      The temperature sensors were not unified in a higher level of abstraction
+ *      structure because the number of sensors is two and it not likely to change.
+ */
+
+#include "temperatureManager.h"
+
+/*
+ * Temperature sensor gain in Volts/degC
+ */
+#define TEMPERATURE_SENSOR_GAIN 0.01
```

Comment:
I have created i define in Constants.h called TEMP_SENSOR_GAIN 0.01, so we should use the same¿
its already updated in the develop branch¿¿¿¿